src/test/java/org/broadinstitute/hellbender/utils/variant/writers/GVCFWriterUnitTest.java

```diff
@@ -478,4 +499,82 @@ private static void assertGVCFIsParseableAndVariantsMatch(File variantFile, List
         }
 
     }
+
+    @Test
+    public void testAgainstExampleGVCF() throws IOException {
+        final List<Integer> gqPartitions = Arrays.asList(1, 10, 20, 30, 40, 50, 60);
+        final File comparisonFile = new File("src/test/resources/org/broadinstitute/hellbender/utils/variant/writers/small.g.vcf");
+        final File outputFile =  new File("tmp.g.vcf");//createTempFile("generated", ".gvcf");
```

Comment:
woops.  forgot to switch to the temp file after testing with a file I could find easily. fixed